cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesGroupControllerTestIT.java

```diff
@@ -516,7 +521,7 @@ void test_add_assigned_locs() throws Exception {
             .when()
             .redirects().follow(true)
             .redirects().max(3)
-            .delete("/timeseries/group/" + group.getId())
+            .delete("/timeseries/group/{group}",group.getId())
```

Comment:
I like the curly brace replacement, we should do that everywhere.

Reply:
Yeah, I discovered that a while ago, the primary thing it does is change how the Http Request/Response is pretty printed.

Sadly it doesn't fix the issue with "/" in the actual name like for clobs and blobs. (And that's still broken in Javalin. supposed to support it but the processing code doesn't quite work right. Figured I wouldn't bother about it until we can move to javalin 5 though.)